go.mod

```diff
@@ -33,12 +33,12 @@ require (
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2
 	go.etcd.io/bbolt v1.3.8
 	go.uber.org/zap v1.10.0
-	golang.org/x/crypto v0.14.0
-	golang.org/x/net v0.17.0
-	golang.org/x/sync v0.3.0
-	golang.org/x/sys v0.13.0
+	golang.org/x/crypto v0.15.0
+	golang.org/x/net v0.18.0
+	golang.org/x/sync v0.5.0
+	golang.org/x/sys v0.14.0
 	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
-	google.golang.org/grpc v1.58.3
+	google.golang.org/grpc v1.59.0
```

Comment:
Usually we don't proactively bump dependency in a stable release unless there is any CVE or critical/major issue. Also we should do it in a separate PR if we decided to do it.

But I won't insist on it this time if there is no any objection from other maintainers and grpc-go expert/maintainer. @dfawley

Also interesting .... I see that the grpc-go's tag `v1.59.0` was created on Oct 17, 2023, but I do not see grpc-go team explicitly releases it, as I do not see the release in https://github.com/grpc/grpc-go/releases @dfawley is there any reason that you intentionally did not release grpc-go v1.59.0?

Reply:
Removed the change since it is not necessary for this PR.